src/common/transport/transport-client.js

```diff
@@ -0,0 +1,80 @@
+import { TransportClientError } from '../errors';
+import rp from 'request-promise';
```

Comment:
* Dropping Babel: all for it. There were just a few things that Node didn't support which is why I went with babel but they're more syntax preferences than anything. Shouldn't be hard to rip out
* request-promise-native: again, should be pretty easy to replace. There's only one spot calling it and I don't have any strong feelings for Bluebird. I actually like the idea of sticking with native promises.
* Do the Domo APIs have CORS enabled that this SDK could work on the browser? I like that idea and I'm happy to make the changes. I wasn't aware that was possible if that is the case though.